sdk/tests/tutorials/marketdata/test_quotes.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from datetime import datetime, timedelta
+
+import pytz as pytz
+
+import lusid
+import lusid.models as models
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from utilities.credentials_source import CredentialsSource
+from utilities.test_data_utilities import TestDataUtilities
+
+
+class Quotes(unittest.TestCase):
+    quotes_api = None
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.quotes_api = lusid.QuotesApi(api_client)
+
+    def test_add_quote(self):
+
+        request = models.UpsertQuoteRequest(
+            quote_id=models.QuoteId(
+                provider="DataScope",
+                price_source="USDRC",
```

Comment:
Change all price_source to be "" as this is the new default.

Reply:
Remove `price_source`